compiler/tools/fpp-check/test/redef/tests.sh

```diff
@@ -17,6 +17,7 @@ module_component_instance
 module_constant
 module_enum
 module_port
+module_statemachine
```

Comment:
I see component_statemachine.fpp but no test or ref for it. Let's be consistent: statemachine --> state_machine.

Reply:
Fixed